webapp/channels/src/components/post_emoji/post_emoji.tsx

```diff
@@ -22,14 +24,22 @@ const PostEmoji = ({name, imageUrl}: Props) => {
     }
 
     return (
-        <span
-            alt={emojiText}
-            className='emoticon'
+        <WithTooltip 
+            id='postEmoji__tooltip'
             title={emojiText}
-            style={{backgroundImage: backgroundImageUrl}}
+            emoji={name}
+            emojiStyle='large'
+            placement='top'
         >
-            {emojiText}
-        </span>
+            <span
+                alt={emojiText}
+                className='emoticon'
+                data-testid={`postEmoji.${emojiText}`}
```

Comment:
Why the change on the test id?

Reply:
As `title` attribute in the span tag renders the default tooltip which is not required in this case. `testid` added here to find the required element in unit tests.